### ISSUE TITLE
move all field values, help text, and attribute descriptions to a config file

### ISSUE THREAD

**Issue by jmcmichael** [genome/civic-client#236]
These are repeated in several places throughout the app, so should be refactored into a single location.

**Comment by malachig** [comment]
Was this started?  Is this the purpose of this component:

`src/components/services/ConfigService.js`

**Comment by jmcmichael** [comment]
Yes, ConfigService can be injected into any component. Its intended as a global storage for form/tip/ universally to hold tool tip text, menus, etc. 

In in the evidenceSummary component, configService is injected here:

https://github.com/genome/civic-client/blob/master/src/app/views/events/evidence/summary/evidenceSummary.js#L22

and assigned to its scope here:

https://github.com/genome/civic-client/blob/master/src/app/views/events/evidence/summary/evidenceSummary.js#L27

to be used in its template, e.g. here:

https://github.com/genome/civic-client/blob/master/src/app/views/events/evidence/summary/evidenceSummary.tpl.html#L19

This same pattern can be used to unify the configuration for the various Add/Edit views.

**Comment by jmcmichael** [comment]
This was pretty much done but it's not as clean as it could be - the new Assertions features have help text specified directly in the field template options. These need to be moved their own Assertions section of ConfigService.
